crypto/bio/b_print.c

```diff
@@ -806,10 +806,12 @@ doapr_outch(char **sbuffer,
             char **buffer, size_t *currlen, size_t *maxlen, int c)
 {
     /* If we haven't at least one buffer, someone has doe a big booboo */
-    OPENSSL_assert(*sbuffer != NULL || buffer != NULL);
+    if (!ossl_assert(*sbuffer != NULL || buffer != NULL))
```

Comment:
That fact is not trivially obvious. This function has many call sites. You'd have to go through every one of them to confirm that it's true (which I haven't done). The assert has the added benefit of documenting the expectation, that at least one buffer is non-NULL.